### PR TITLE
test: create functions in network policy specific file and let tests call it.

### PR DESCRIPTION
**Pull request by csfmomo**
**Reason for Change**:
Functions created in network policy specific file for tests to consume. Increase code reusablity.

**Requirements**:

- [X ] uses [conventional commit messages](https://www.conventionalcommits.org/)
        test: Testing 💚 
- [ ] includes documentation
- [ ] adds unit tests
- [X ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: test/e2e/kubernetes/namespace/namespace.go [hunk 1]
```diff
@@ -41,8 +41,18 @@ func CreateIfNotExist(name string) (*Namespace, error) {
 	n, err := Get(name)
```
Comment (jackfrancis) [inline test/e2e/kubernetes/namespace/namespace.go]: Can we rename this function to `CreateDeleteIfExists` to reflect the new behavior?

File: test/e2e/kubernetes/kubernetes_test.go [hunk 2]
```diff
@@ -704,6 +704,13 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			By("Ensuring that we have functional DNS resolution from a linux container")
 			validateDNSLinuxName := "validate-dns-linux"
 			validateDNSLinuxNamespace := "default"
+			// Create a kubectl version check to avoid api version mismatch which due to deployment failure in next step
```
Comment (jackfrancis) [inline test/e2e/kubernetes/kubernetes_test.go]: FYI, this is fine as an informational check, but the `k` util is already designed to use the version of `kubectl` that the API returns in a call to the cluster. Is that not working in your experience?

Comment (csfmomo) [reply test/e2e/kubernetes/kubernetes_test.go]: In my local test with out ssh enabled, I found my kubectl version comes from https://github.com/Azure/aks-engine/blob/master/scripts/k#L40
It's usually older than test cluster's version which result in k create * failure. 
I didn't experience auto cluster call and match in local so I add this check. I have to manually override it. However, from pipeline validation, it seems kubectl version can match.

File: test/e2e/kubernetes/namespace/namespace.go (hunk 1 above)
Comment (csfmomo) [reply test/e2e/kubernetes/namespace/namespace.go]: Sure, let me rename it.

**Pull request merged**